cdbot/cogs/cyber.py

```diff
@@ -360,6 +360,74 @@ def __init__(self, bot: Bot):
         await ctx.send(f"There are {len(game_r.members)} that have completed Cyberstart Game. Out of them, "
                        f"{len(true_r.members)} have also completed Essentials and Assess.")
 
+    @command()
+    async def elitecountdown(self, ctx: Context, event_name: str = None, event_level: str = None):
+        """
+        Gets the time until elite, and shows in an embed
+        """
+        valid_elite_locations = ["London", "Birmingham", "Lancaster "]
+
+        # Validate and normalise inputs
+        event_name = event_name.capitalize()
+        if event_name:
+            if not (event_name in valid_elite_locations):
```

Comment:
This as well cause you could have
```python
"London": "London"
```
in `ELITE_ALISES`

Reply:
Personally I just think that is more confusing?

Reply:
Ask others but IMHO it's better and removes a useless check